### ISSUE TITLE
v1.0.5: problem with GraphAligner v1.0.13?

### ISSUE THREAD

**Issue by ptrebert** [maickrau/GraphAligner#44]
Hi,
I updated to MBG v1.0.5 mostly because of the comment about lower memory requirements, and now I am getting thousands of assertion errors during ONT alignment of the following type...

```
src/ReadCorrection.cpp:44: Assertion 'i == 0 || corrections[i].startIndex >= corrections[i-1].startIndex' failed.
```
...until GA exits with a segfault for a particular read. Following your statement that maickrau/MBG#1 has been fixed in v1.0.4+, I am using the MBG graphs as-is (no parameter `--blunt` and no post-processing with vg). Since I did not update GA, I decided to open the issue here, because the problem seems to be caused by a change in MBG; if you are unhappy with that, I can copy the issue over to the GA repo.

I added all of the above data to the Globus share (plus log files for graph building and alignment), just let me know the email address to send the Globus share details.

Best,
Peter

**Comment by maickrau** [comment]
The assertion is a bug in GraphAligner v1.0.13 which is fixed in the latest commit in master but isn't in bioconda yet.

**Comment by maickrau** [comment]
This was fixed with version 1.0.14 some time ago
